recordium/main.py

```diff
@@ -249,6 +258,13 @@ def closeEvent(self, event):
         if messages_to_remove:
             self._storage.delete_elements(messages_to_remove)
             self._systray.set_message_number()
+
+        # saves column order configuration
+        col_order = []
+        for col in range(0, self.columnCount()):
+            col_order.append(str(self.visualColumn(col)))
+        config.set(config.COL_ORDER, col_order)
+        config.save()
```

Comment:
We shouldn't be touching disk *always* here. 

Most probably, 99.9999999% of the cases the user will not change this config, so hitting disk everytime seems like a huge waste here.

What you could do is to compare the current order with the configured one; if different, save it. Other thing that you could do (but didn't explored it myself) would be to hook into a signal for the "columns order changed" situation, have an internal flag about this, and if that flag is set, save the config)